src/acftbias/generate_aircraft_bias_csv.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/python3
+
+import csv
+
+# csv_write: fill a CSV file with coordinate-variable and draw-variable data for JEDI DrawValueFromFile
+#
+# INPUTS
+#
+# csvFileName: name of a write-allowable file for CSV output, will be generated or clobbered (string)
+# coefFileName: name of a read-allowable space-delimited GDAS bias correction file (string)
+# coordVariableName: <Group>/<name> of coordinate variable (string)
+# coordVariableType: variable-type for coordinate variable (string)
+# coordVariableColumn: column-number for coordinate variable in coefFile (integer)
+# drawVariableName: <Group>/<name> of draw variable (string)
+# drawVariableType: variable-type for draw variable (string)
+# drawVariableColumn: column-number for draw variable in coefFile (integer)
+#
+# OUTPUTS
+#
+# no returned variables, but CSV file is properly written and formatted for JEDI DrawValueFromFile
+#
+# DEPENDENCIES
+#
+# csv
+#
+def csv_write(csvFileName, coefFileName, coordVariableName, coordVariableType, coordVariableColumn,
+              drawVariableName, drawVariableType, drawVariableColumn):
+    import csv
+    csvFile = open(csvFileName, 'w')
+    cwrite = csv.writer(csvFile)
+    # write 1st-line header for coordVariableName, drawVariableName
+    cwrite.writerow([coordVariableName, drawVariableName])
+    # write 2nd-line header for coordVariableType, drawVariableType
+    cwrite.writerow([coordVariableType, drawVariableType])
+    # write first data line for missing value placeholder '_' to set draw for missing coordinates to a
+    # selected missing value. For aircraft bias correction, setting these coefficient values to 0. will
+    # effectively skip bias correction for these data.
+    cwrite.writerow(['_', '0.'])
+    # write remaining lines from coefFile columns specified by coordVariableColumn, drawVariableColumn
+    # NOTE: represent a tail-number coordinate variable as a left-justified space-padded string by
+    #       using the str.ljust() command
+    coefFile = open(coefFileName, 'r')
+    for line in coefFile:
+        cwrite.writerow([line.split()[coordVariableColumn].ljust(8,' '), line.split()[drawVariableColumn]])
+    return
+    # close files
+    csvFile.close()
+    coefFileName.close()
+#
+# begin
+#
+if __name__ == "__main__":
+    # define GDAS aircraft bias file name for read-only
+    coefFileName = 'gdas.t18z.abias_air'
```

Comment:
what about making these 4 variables defined with `argparse` rather than hard coded

Reply:
I can do that, I could also slim this down to a single variable output and if someone wanted to generate all 3 CSVs they could run it 3 times with explicit `argparse` inputs

Reply:
fixed